src/ansys/dpf/core/server_types.py

```diff
@@ -1221,14 +1210,15 @@ def local_server(self, val):
 
     def shutdown(self):
         if self._own_process and self.live:
+            info = self.info # initializing the info variable (giving access to ip and port): this can be required if start_local_server is called afterwards
```

Comment:
```suggestion
            _ = self.info # initializing the info variable (giving access to ip and port): this can be required if start_local_server is called afterwards
```